### ISSUE TITLE
Support action handlers

### ISSUE THREAD

**Issue by bomberstudios** [skpm/skpm#56]
My `manifest.json` uses action handlers like this:

```
    …
    "commands": [
        {
            "name": "Artboard Manager",
            "identifier": "artboardmanager",
            "script": "./artboard-manager.js",
            "handlers" : {
                "actions": {
                    "ArtboardChanged.finish": "ArtboardChanged",
                    "LayersMoved.finish": "LayersMoved",
                    "Duplicate.finish": "Duplicate",
                    "HandlerLostFocus": "Resize",
                    "ResizeArtboardToFit": "ResizeArtboardToFit"
                }
            }
        },
    …
```

Currently, building a plugin will only create a handler for the `onRun` event. As a workaround, I am creating a new dummy command for each action I want to trigger:

```
        {
            "script" : "./artboard-manager.js",
            "name" : "Artboard Manager - ArtboardChanged",
            "handler" : "ArtboardChanged",
            "identifier" : "artboardmanager.artboardchanged"
        },
        {
            "script" : "./artboard-manager.js",
            "name" : "Artboard Manager - LayersMoved",
            "handler" : "LayersMoved",
            "identifier" : "artboardmanager.layersmoved"
        },
        {
            "script" : "./artboard-manager.js",
            "name" : "Artboard Manager - Duplicate",
            "handler" : "Duplicate",
            "identifier" : "artboardmanager.duplicate"
        },
        {
            "script" : "./artboard-manager.js",
            "name" : "Artboard Manager - Resize",
            "handler" : "Resize",
            "identifier" : "artboardmanager.resize"
        },
        {
            "script" : "./artboard-manager.js",
            "name" : "Artboard Manager - ResizeArtboardToFit",
            "handler" : "ResizeArtboardToFit",
            "identifier" : "artboardmanager.ResizeArtboardToFit"
        }
```

but it would be nice if skpm supported action handlers out of the box.

**Comment by mathieudutour** [comment]
I'm not touching the handlers field so you should be able to use it.

in your `artboard-manager.js`, you can export multiple handlers:
```js
export function ArtboardChanged (context) {
...
}

export function LayersMoved (context) {
...
}
```

**Comment by bomberstudios** [comment]
Yes, I can define those, but unless I add my workaround they are not registered in the compiled JS.

Without the workaround, my JS file looks like this at EOF:

```
that['onRun'] = run.bind(this, 'default');
```

and with the workaround, it looks like this:

```
that['onRun'] = run.bind(this, 'default');
that['ArrangeArtboards'] = run.bind(this, 'ArrangeArtboards');
that['ArtboardChanged'] = run.bind(this, 'ArtboardChanged');
that['LayersMoved'] = run.bind(this, 'LayersMoved');
that['Duplicate'] = run.bind(this, 'Duplicate');
that['Resize'] = run.bind(this, 'Resize');
that['ResizeArtboardToFit'] = run.bind(this, 'ResizeArtboardToFit')
```

Hope that makes it clearer (check https://github.com/bomberstudios/artboard-manager for the actual code)

**Comment by mathieudutour** [comment]
How I didn't know it was possible to write `handlers.actions`. Guess I didn't really look.

Fixed in 0.10.3 anyway

**Comment by bomberstudios** [comment]
Awesome, thanks!
